orc8r/cloud/helm/orc8r/charts/nms/templates/helpers/pod_anti_affinity.tpl

```diff
@@ -0,0 +1,43 @@
+{{/*
```

Comment:
@xjtian  this is required for pod advanced scheduling . in some cases if we need to scale pod-a on the same node where already that pod is running then this will help, also vice-versa.

Reply:
same as the resource template and the service accounts, let's add this when we need it, to the helpers toolbox. at the scale of our typical upcoming deployments this seems unnecessary.